product_abc_classification_base/models/abc_classification_product_level.py

```diff
@@ -140,4 +140,21 @@ def create(self, vals):
             # at creation the manual level is set to the same value as the
             # computed one
             vals["manual_level_id"] = vals["computed_level_id"]
+        
+        profile = self.env["abc.classification.profile"].browse(vals['profile_id'])
+        if profile.auto_apply_computed_value and "computed_level_id" in vals:
+            vals["manual_level_id"] = vals["computed_level_id"]
```

Comment:
@lmarion-source Attention, au moment du call à create, les containtes ne sont pas vérifiées donc "profile_id" pourrait ne pas être dans vals.... Dans un tel cas, le code va échoué avant que le système ait pu répondre avec un message approprié pour la valeur manquante...
```suggestion
        if "profile_id" in vals:
            profile = self.env["abc.classification.profile"].browse(vals['profile_id'])
            if profile.auto_apply_computed_value and "computed_level_id" in vals:
                vals["manual_level_id"] = vals["computed_level_id"]
```

Reply:
@lmarion-source je vais finir la PR